### PR TITLE
Fix StackOverflowError crash in Fiery Armor

### PR DESCRIPTION
**Pull request by Sleepy-Horse**
There was a StackOverflowError crash in rendering when wearing Fiery Armor. I have fixed it.

### CODE REVIEW

**Review by GizmoTheMoonPig — CHANGES_REQUESTED** [review]
This is not the way to fix this. 

This PR removes the fullbright property of fiery armor as a side effect of deleting the model class. The proper fix here is just to add the `color` parameter to the super call.

**Comment by Sleepy-Horse** [comment]
Here's the screenshot
image

File: src/main/java/twilightforest/client/model/armor/FieryArmorModel.java [hunk 1]
```diff
@@ -12,7 +12,7 @@ public FieryArmorModel(ModelPart part) {
 
 	@Override
 	public void renderToBuffer(PoseStack stack, VertexConsumer builder, int light, int overlay, int color) {
-		super.renderToBuffer(stack, builder, 0xF000F0, overlay);
+		super.renderToBuffer(stack, builder, 0xF000F0, overlay, 654311423);
```
Comment (Tamaized) [inline src/main/java/twilightforest/client/model/armor/FieryArmorModel.java]: Why 0x26FFFFFF ? Shouldn't it be 0xFFFFFFFF ?
The super uses -1 which is 0xFFFFFFFF.

Comment (Tamaized) [reply src/main/java/twilightforest/client/model/armor/FieryArmorModel.java]: Also use hex for colors as its much more readable than decimal.

**Pull request merged**